.github/workflows/ci.yml

```diff
@@ -40,3 +40,26 @@ jobs:
         with:
           lua-version: ${{ matrix.version }}
           rockspec: ${{ matrix.package }}
+
+
+  hello-haproxy:
+    runs-on: ubuntu-latest
+    env:
+      LD_SDK_KEY: "foo"
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build hello-haproxy image
+        run: |
+          docker build -t launchdarkly:hello-haproxy -f ./examples/hello-haproxy/Dockerfile .
+      - name: Run hello-haproxy container in background
+        # The sleep is to give haproxy some time to start-up.
+        run: |
+          docker run -dit --rm --name hello-haproxy -p 8123:8123 --env LD_SDK_KEY="$LD_SDK_KEY" launchdarkly:hello-haproxy
+          sleep 5
```

Comment:
This is probably fine, but if it ends up being a problem you could instead just use curl with something like `--retry 10 --retry-connrefused --retry-delay 1`

Reply:
That's probably more robust.